### PR TITLE
chore: add release notes to version button in status bar

### LINKED ISSUE #8521: Make Release Note available from the release version displayed in the status bar
### Is your enhancement related to a problem? Please describe

As a user I would like to be able to view the latest release note of Podman Desktop

### Describe the solution you'd like

Display the release notes behind the release version button from the status bar

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by SoniaSandler**
Signed-off-by: Sonia Sandler <ssandler@redhat.com>

### What does this PR do?
Adds `View release notes` button to the status bar version message box. 

### Screenshot / video of UI
(will work like this after https://github.com/containers/podman-desktop/pull/8753 is merged)

[Screencast from 2024-09-23 21-26-30.webm](https://github.com/user-attachments/assets/ff8cf715-a9af-41a2-88dd-d9178fbdbfb0)

### What issues does this PR fix or reference?
Closes https://github.com/containers/podman-desktop/issues/8521

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Comment by slemeur** [comment]
thank you @SoniaSandler :)

File: packages/main/src/plugin/updater.ts [hunk 1]
```diff
@@ -163,8 +170,10 @@ export class Updater {
         buttons: buttons,
         cancelId: 1,
       });
-      if (result.response === 2) {
+      if (result.response === 3) {
         this.updateConfigurationValue('never');
+      } else if (result.response === 1) {
+        console.log('await this.openReleaseNotes(updateVersion)');
```
Comment (axel7083) [inline packages/main/src/plugin/updater.ts]: ?

File: packages/main/src/plugin/updater.ts [hunk 2]
```diff
@@ -99,27 +101,32 @@ export class Updater {
...
+      const result = await this.messageBox.showMessageBox({
         type: 'info',
         title: 'Version',
         message: `Using version ${this.#currentVersion}`,
         detail: detailMessage,
+        buttons: ['View release notes'],
       });
+      if (result.response === 0) {
+        await this.configurationRegistry.updateConfigurationValue(`releaseNotesBanner.show.${app.getVersion()}`, true);
+        console.log(`this.apiSender.send('show-release-notes');`);
```
Comment (axel7083) [inline packages/main/src/plugin/updater.ts]: ?

**Review by axel7083 — COMMENTED** [review]
Could we have a screenshots or video of the change included in the PR description ? Thanks!

File: packages/main/src/plugin/updater.ts (hunk 1 above)
Comment (SoniaSandler) [reply packages/main/src/plugin/updater.ts]: @axel7083 the `console.log` lines I added are necessary for this PR, but they don't work until https://github.com/containers/podman-desktop/pull/8753 is merged. Commenting them gives an error with the linter, so I just put them in `console.log`. rn this PR is not functional until https://github.com/containers/podman-desktop/pull/8753 is merged

**Comment by feloy** [comment]
There is no effect when the user clicks on the "View release notes" button when the user is not on the Dashboard page. Would we want to redirect it to the dashboard (but I'm not sure it is a good idea)?

**Comment by SoniaSandler** [comment]
@feloy if the release notes were in their own separate page than yes, but since they are in the dashboard page along with other things, I'm also not sure if the users would want to be taken automatically to the dashboard page from wherever they are in the app.
if it is possible, maybe I could check where we are currently on the app (dashboard vs somewhere else), and then if it is not the dashboard page, open the `podman-desktop.io` or GitHub notes in the browser

**Review by jeffmaury — COMMENTED** [review]
Can we hide the button if the release notes is not available. It tried with 0.9.0 and 1.13.0-next but pressing the button leads no NOP

**Comment by SoniaSandler** [comment]
@feloy @jeffmaury After some discussion with Jeff, we decided to leave it as it is for now, and if needed, open a follow up to modify the behavior of the button.
For now, the expected behavior is if the release notes in the dashboard page are closed/hidden, clicking the `View release notes` button will show them, which means from any non-dashboard page, it will look like NOP.
If the release notes are already visible, it will also look like NOP.
If there is an update available (mac or windows), the `View release notes` will open the `podman-desktop.io` release notes or GitHub release summary of the new version in the browser.

**Review by benoitf — APPROVED** [review]

**Review by deboer-tim — APPROVED** [review]

**Pull request merged**